pepys_admin/view_data_cli.py

```diff
@@ -0,0 +1,143 @@
+import cmd
+
+from iterfzf import iterfzf
+from prompt_toolkit import HTML, prompt
+from prompt_toolkit.lexers import PygmentsLexer
+from pygments.lexers.sql import SqlLexer
+from sqlalchemy import inspect
+from sqlalchemy.ext.associationproxy import AssociationProxy
+from sqlalchemy.orm import RelationshipProperty, class_mapper, load_only
+from tabulate import tabulate
+
+
+def bottom_toolbar():
+    return HTML('Press <b><style bg="ansired">ESC+Enter</style></b> to exit!')
```

Comment:
@BarisSari - when do we see this toolbar get displayed?

Reply:
It should be displayed when you select `run sql` option.